client/my-sites/sharing/connections/services-group.jsx

```diff
@@ -1,123 +1,87 @@
  */
-var NUMBER_OF_PLACEHOLDERS = 4;
-
-module.exports = React.createClass( {
-	displayName: 'SharingServicesGroup',
-
-	propTypes: {
-		site: React.PropTypes.object,
-		user: React.PropTypes.object,
-		connections: React.PropTypes.object,
-		onAddConnection: React.PropTypes.func,
-		onRemoveConnection: React.PropTypes.func,
-		onRefreshConnection: React.PropTypes.func,
-		onToggleSitewideConnection: React.PropTypes.func,
-		initialized: React.PropTypes.bool,
-		services: React.PropTypes.array,
-		title: React.PropTypes.string.isRequired,
-		description: React.PropTypes.string
-	},
-
-	getDefaultProps: function() {
-		return {
-			onAddConnection: function() {},
-			onRemoveConnection: function() {},
-			onRefreshConnection: function() {},
-			onToggleSitewideConnection: function() {},
-			initialized: false
-		};
-	},
-
-	getEligibleServices: function() {
-		const { site, services } = this.props;
-
-		if ( ! site ) {
-			return services;
-		}
-
-		return services.filter( function( service ) {
-			// Omit if the site is Jetpack and service doesn't support Jetpack
-			if ( site.jetpack && ! service.jetpack_support ) {
-				return false;
-			}
-
-			// Omit if Jetpack module not activated
-			if ( site.jetpack && service.jetpack_module_required &&
-					! site.isModuleActive( service.jetpack_module_required ) ) {
-				return false;
-			}
-
-			// Omit if service is settings-oriented and user cannot manage
-			if ( 'eventbrite' === service.name && ! utils.userCan( 'manage_options', site ) ) {
-				return false;
-			}
-
-			// Omit if Publicize service and user cannot publish
-			if ( 'publicize' === service.type && ! utils.userCan( 'publish_posts', site ) ) {
-				return false;
-			}
-
-			return true;
-		} );
-	},
-
-	renderService: function( service ) {
+const NUMBER_OF_PLACEHOLDERS = 4;
+
+class SharingServicesGroup extends Component {
+
+	static propTypes = {
+		site: PropTypes.object,
+		user: PropTypes.object,
+		connections: PropTypes.object,
+		onAddConnection: PropTypes.func,
+		onRemoveConnection: PropTypes.func,
+		onRefreshConnection: PropTypes.func,
+		onToggleSitewideConnection: PropTypes.func,
+		initialized: PropTypes.bool,
+		services: PropTypes.array,
+		title: PropTypes.string.isRequired,
+		description: PropTypes.string
+	};
+
+	static defaultProps = {
+		onAddConnection: function() {},
+		onRemoveConnection: function() {},
+		onRefreshConnection: function() {},
+		onToggleSitewideConnection: function() {},
+		initialized: false
+	};
+
+	renderService( service ) {
 		return <Service
-			key={ service.name }
-			site={ this.props.site }
-			user={ this.props.user }
+			key={ service.ID }
 			service={ service }
-			connections={ this.props.connections }
-			onAddConnection={ this.props.onAddConnection }
-			onRemoveConnection={ this.props.onRemoveConnection }
-			onRefreshConnection={ this.props.onRefreshConnection }
-			onToggleSitewideConnection={ this.props.onToggleSitewideConnection } />;
-	},
+			{ ...this.props } />;
```

Comment:
Arguably the prior explicitness would be preferable to a spread, since otherwise it becomes less transparent what is being passed and why.